src/components/layout.scss

```diff
@@ -4,6 +4,10 @@
   flex-wrap: wrap;
 }
 
+.programs .content-container {
+  padding: 100px 84px 0;
+}
+
```

Comment:
What is this for? I'm not sure it is doing anything.

Reply:
@LayaTaal That one is just removing the padding on the bottom of the content container on the programs page. I'll update that the set the padding-bottom so it's clearer what it's doing.